SDPi_Supplement/asciidoc/volume1/tf1-ch-b-ref-standards-conformance.adoc

```diff
@@ -21,11 +21,14 @@ NOTE: the following section was moved from the mothership document to here for i
 [bibliography]
 === Referenced Standards
 
-* [[[ieee_11073_10207_2017,IEEE 11073-10207:2017]]] IEEE 11073-10207-2017, Health informatics — Point-of-care medical device communication — Part 10207: Domain Information and Service Model for Service-Oriented Point-of-Care Medical Device Communication, 2018-02, available at https://standards.ieee.org/standard/11073-10207-2017.html.
+##ADD REFERENCES FROM GLOSSARY
+##
 
-* [[[ws_discovery_2009, WS-Discovery:2009]]] Web Services Dynamic Discovery (WS-Discovery) Version 1.1, OASIS Standard, 1 July 2009, available at http://docs.oasis-open.org/ws-dd/discovery/1.1/wsdd-discovery-1.1-spec.html
+* [[[ieee_11073_10207_2017,IEEE 11073-10207:2017]]] IEEE 11073-10207-2017, Health informatics — Point-of-care medical device communication — Part 10207: Domain Information and Service Model for Service-Oriented Point-of-Care Medical Device Communication, 2018-02, available at https://standards.ieee.org/standard/11073-10207-2017.html.  footnote:ieee_permission[]
```

Comment:
Is there a reason for the re-ordering? Does it follow a specific rule?

Reply:
1st - IHE profiles / TF's have no place for including references such as this, other than the various standards conformance sections associated with profiles and transactions.  The only change should have been adding the footnote: macro.  Was the actual sequencing of 26 and 27 changed?  If so ... it wasn't intentional!!!